apps/www/pages/security.mdx

```diff
@@ -122,6 +122,16 @@ In addition to internal security reviews, we use various tools to scan our code
 
 </Section>
 
+<Section icon={<ShieldCheckIcon strokeWidth={1}/>}>
+
+### DDoS Protection
+
+Supabase combats Distributed Denial of Service attacks in several ways to mitigate resource abuse and prevent runaway bills.
+
+In addition to protection at the Cloudflare level, we employ fail2ban on instances and [customizable rate limiting and abuse prevention](/docs/guides/platform/going-into-prod#rate-limiting-resource-allocation--abuse-prevention) safeguards at the API level such as CAPTCHA for auth endpoints. Users can set [spend caps](/docs/guides/platform/spend-cap) to prevent surprise bills.
```

Comment:
```suggestion
In addition to protection at the Content Delivery Network level via Cloudflare, we employ [fail2ban](https://github.com/fail2ban/fail2ban) to prevent brute force logins. Users can set up [customizable rate limits and CAPTCHA](/docs/guides/platform/going-into-prod#rate-limiting-resource-allocation--abuse-prevention) for critical API routes and [spend caps](/docs/guides/platform/spend-cap) to prevent surprise bills.
```